piq/clip_iqa.py

```diff
@@ -0,0 +1,113 @@
+r"""This module implements CLIP-IQA metric in PyTorch.
+
+The metric is proposed in:
+"Exploring CLIP for Assessing the Look and Feel of Images"
+by Jianyi Wang, Kelvin C.K. Chan and Chen Change Loy.
+AAAI 2023.
+https://arxiv.org/abs/2207.12396
+
+This implementation is inspired by the offisial implementation but avoids using MMCV and MMEDIT libraries.
+Ref url: https://github.com/IceClear/CLIP-IQA
+"""
+import os
+import torch
+
+from torch.nn.modules.loss import _Loss
+from typing import Union
+
+from piq.feature_extractors.clip import load
```

Comment:
I would prefer importing a higher level subpackage so that it's obvious in the code below from where this `load` function came. And maybe rename it to better reflect what exactly it's loading?